tests/decorators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -108,8 +108,10 @@ import { observable } from "mobx";
 
 @observer
 class OrderLine {
-  @observable price: number = 0;
-  @observable amount: number = 1;
+  @observable
+  price: number = 0;
+  @observable
```

Comment:
It felt a bit odd to not have an empty line between these, but since we preserve user-entered empty lines, I think it's fine (see also the type orm snapshot)